README.md

```diff
@@ -2,36 +2,50 @@
 
 ## Overview
 
-A brief description. Be clear about the purpose of the resolver,
-its capabilities and limitations.
+A custom filter for Sceptre for unquoting resolvers appearing in var files.
```

Comment:
A custom filter/ A custom Jinja filter

Also can you please provide some information on the motivation behind the creation of this filter?  For example, what use case(s) is this filter solving?

Reply:
added